.babelrc

```diff
@@ -12,6 +12,19 @@
           "transform-regenerator"
         ]
       }
+    ],
```

Comment:
Remove all the 4.3 related babel configuration

Reply:
done